docs/docs/configuration/cache.md

```diff
@@ -96,11 +96,11 @@ $ trivy server --cache-backend redis://localhost:6379 \
   --redis-key /path/to/key.pem
 ```
 
-[trivy-db]: ./db.md#vulnerability-database
-[trivy-java-db]: ./db.md#java-index-database
+[trivy-db]: ./db.md
+[trivy-java-db]: ./db.md
 [misconf-checks]: ../scanner/misconfiguration/check/builtin.md
 [boltdb]: https://github.com/etcd-io/bbolt
-[parallel-run]: https://aquasecurity.github.io/trivy/v0.52/docs/references/troubleshooting/#running-in-parallel-takes-same-time-as-series-run
+[parallel-run]: https://trivy.dev/v0.52/docs/references/troubleshooting/#running-in-parallel-takes-same-time-as-series-run
```

Comment:
```suggestion
[parallel-run]: https://trivy.dev/latest/docs/references/troubleshooting/#running-in-parallel-takes-same-time-as-series-run
```

Reply:
thanks, I changed to `{{ git.tag }}` instead

Reply:
Looks like we can just use `latest` (like for other links).
Do you see a reason why we need to use `{{ git.tag }}` here?